### DISCUSSION TITLE
Store data in firestore locally on device only

### DISCUSSION THREAD

**Discussion by MadsStolt** [firebase/flutterfire#10428] [Q&A] [answered]
Does anyone know of a way of using the firestore as a local database only? I am already using firestore for storing shared data in the cloud database, however some of my data is only used locally, so it would be awesome if I were able to add a collection locally on the device only. Does any know if this is possible, or should I add another db like sqlite for this use case?

**Comment by rusudinu** [comment, marked as answer]
No, as soon as the device gets an internet connection, it will sync with the cloud. You should use shared preferences if you have to store simple data (works pretty well if you don't need to have search, or other DB-specific queries and you only want to store a list of objects, etc), or indeed use another db like sqlite to store data locally.
